plugins/utilities/easy_connect.py

```diff
@@ -458,7 +458,23 @@ def popup_menu_selected_choice(self, window: popup.PopupMenu,
         ba.playsound(ba.getsound('gunCocking'))
 
 
+def is_game_version_lower_than(version):
+    """
+    Returns a boolean value indicating whether the current game
+    version is lower than the passed version. Useful for addressing
+    any breaking changes within game versions.
+    """
+    game_version = tuple(map(int, ba.app.version.split(".")))
+    version = tuple(map(int, version.split(".")))
+    return game_version < version
+
+
 def replace():
+    global ba_internal
+    if is_game_version_lower_than("1.7.7"):
+        ba_internal = _ba
+    else:
+        ba_internal = ba.internal

```

Comment:
I think a more readable way would be to define this at the top instead of declaring a global variable here.

Reply:
ok sure, I will make the changes soon.